docs/running/overview.rst

```diff
@@ -2,16 +2,73 @@
 
 Overview
 ========
-This section describes how to run Toil in various environments, including locally, in the cloud, and with using CWL. 
+This section describes how to run Toil in various environments, including :ref:`locally <fileJobStore>`,
```

Comment:
I would make a slight change to this sentence:

"This section describes how to run Toil in various environments: locally, in the clound and using CWL."